rsocket-examples/src/test/java/io/rsocket/integration/IntegrationTest.java

```diff
@@ -90,6 +93,8 @@ public void startup() {
 
     server =
```

Comment:
I'll test that via more specific tests, e.g. a unit test of the builders that shows the right interceptors are applied.  I think Integration tests are better left for testing end to end scenarios.